.eslintignore

```diff
@@ -1,2 +1,3 @@
 .cache/**/*.js
 public/**/*.js
+src/html.js
```

Comment:
This file is copied from Gatsby, so I think it is in our best interest to keep it as they format it for ease of update in the future